README.md

```diff
@@ -0,0 +1,8 @@
+# install-hook-tutorial
+Learning from Kyrofa - hook-tutorial (Linux is not my speciality) NO SHIT :)
+I came across this code snippet by chance and it caught my attention.  
+
+I am unfamiliar with this 'hook' technique and the way script files can be used.
+So - thank you Kyrofa for the lesson.
```

Comment:
You're welcome! I'll go ahead and close this since it's not really something to be merged, but feel free to log issues if you have questions.